lib/pages/main/tab_wallet_contents.dart

```diff
@@ -280,8 +283,40 @@ class _TabWalletContentsState extends State<TabWalletContents> {
                                 ThemePaddings.smallPadding,
                                 ThemePaddings.normalPadding,
                                 ThemePaddings.miniPadding),
-                            child: Text(l10n.homeHeader,
-                                style: TextStyles.sliverCardPreLabel)))
+                            child: Row(
+                                crossAxisAlignment: CrossAxisAlignment.center,
+                                mainAxisAlignment:
+                                    MainAxisAlignment.spaceBetween,
+                                children: [
+                                  //Header text
+                                  Text(l10n.homeHeader,
+                                      style: TextStyles.sliverCardPreLabel),
+                                  Observer(builder: (context) {
+                                    print("in!");
```

Comment:
We shouldn't use print. 
I guess there is a way to do swizzling for the method (it's called like that in iOS) for the print method or if this is not possible, to add some directive in the analyser to prevent to use it.

Reply:
Fixed